src/ledger/backend/tokencache.ts

```diff
@@ -20,10 +20,31 @@ interface Responder {
 	symbol(): Promise<string>;
 }
 
+export interface TokenCache {
+	readonly typeCache: TokenTypes;
+	readonly holderCache: TokenHolders;
+	setType(tokenAddr: string, ttype: TokenType): void;
```

Comment:
This might be redundant, but I could not decide.

Reply:
I think it makes sense to have a `setType` method. Or what do you mean?

Reply:
It is useful, I just should have removed the `typeCache` and `holderCache` from the interface and this question would not have come up :clown_face: